deploy/smcp-templates/base

```diff
@@ -4,26 +4,68 @@ spec:
   istio:
     global:
       tag: 1.0.2
+      # constrain resources for use in smaller environments
+      defaultResources:
+        requests:
+          cpu: 50m
+          memory: 128Mi
+        limits:
+          cpu: 50m
+          memory: 256Mi
       proxy:
         autoInject: disabled
         dnsRefreshRate: 300s
+        # constrain resources for use in smaller environments
+        resources:
+          requests:
+            cpu: 50m
+            memory: 128Mi
+          limits:
+            cpu: 50m
+            memory: 256Mi
       oauthproxy:
         tag: 4.1
         imagePullPolicy: IfNotPresent
       defaultPodDisruptionBudget:
         enabled: false
+    security:
+      # constrain resources for use in smaller environments
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 100m
+          memory: 256Mi
+    galley:
+      # constrain resources for use in smaller environments
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 100m
+          memory: 256Mi
     pilot:
+      resources:
+        requests:
+          cpu: 100m
+          memory: 128Mi
+        limits:
+          cpu: 100m
+          memory: 256Mi
       env:
         GODEBUG: null
     mixer:
       telemetry:
+        # constrain resources for use in smaller environments
         resources:
           requests:
-            cpu: 100m
-            memory: 1G
+            cpu: 50m
+            memory: 128Mi
           limits:
-            cpu: 500m
-            memory: 4G
+            cpu: 50m
+            memory: 256Mi
```

Comment:
In the Jira, you tested 512 for Mixer, but it says 256 here. Is this correct?

Reply:
telemetry pod is combined of two containers
- istio-proxy (50m, 50m, 128Mi, 256Mi)
- mixer (50m, 50m, 128Mi, 256Mi)

Combined these give pod resources of 100m, 100m, 256Mi, 512Mi